resources-subsystem-fixtures/pom.xml

```diff
@@ -26,4 +26,16 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
@spriadka Please add the same note as in [tooling-deployment/pom.xml](https://github.com/hal/testsuite.next/pull/164/files#diff-56c9e760a257819818b9803c80d29bf92475bb6ed152582eb60c60f0dfbfbefc)